x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -351,35 +349,10 @@ const EventsViewerComponent: React.FC<Props> = ({
               </FullWidthFlexGroup>
             </EventsContainerLoading>
           </>
-        </EventDetailsWidthProvider>
-      ) : null}
-    </StyledEuiPanel>
+        ) : null}
+      </StyledEuiPanel>
+    </InspectButtonContainer>
   );
 };
 
-export const EventsViewer = React.memo(
-  EventsViewerComponent,
-  // eslint-disable-next-line complexity
-  (prevProps, nextProps) =>
```

Comment:
did removing this comparison impact rendering performance?

Reply:
that was before we bring back the old way with a feature flag, now it should be back as it was